### PR TITLE
Feature/cc 0104 make payment and complete order v2

### PR DESCRIPTION
**Pull request by STY1997**
Screen Shot 2022-11-08 at 19 59 51
Screen Shot 2022-11-08 at 20 01 25
Screen Shot 2022-11-08 at 20 02 28
Screen Shot 2022-11-08 at 20 02 46

### CODE REVIEW

**Review by zhuocun — COMMENTED** [review]
LGTM

File: src/pages/payment.tsx [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+              {status === PageStatus.SUCCESS
+                ? "You have successfully paid the deposit. Our staff will contact you soon."
+                : "Your payment could not be processed. Please try again."}
+            </Text>
+            <Box height="0.1rem" background="#D3D3D3" width="calc(100% - 30vw)"></Box>
+            <OrderInfoCard paymentResPage={paymentInfo} />
+          </Flex>
+          <Flex justifyContent="center" alignItems="center" gap="3rem">
+            <Button variant="shareBtn" padding="1rem 1.5rem" fontSize="1.2rem">
+              Check My Order
```
Comment (zhuocun) [inline src/pages/payment.tsx:121]: A minor one, would you please make this "Check My Order" button and the "Try again" button consistent in writing format? :)

**Review by ShawnWang86b — APPROVED** [review]

**Review by Cecilia0109 — APPROVED** [review]
LGTM

File: src/pages/payment.tsx (hunk 1 above)
Comment (STY1997) [reply src/pages/payment.tsx:121]: > A minor one, would you please make this "Check My Order" button and the "Try again" button consistent in writing format? :)

fixed

**Pull request merged**